config/api.yml

```diff
@@ -742,6 +742,25 @@
       :post:
       - :name: assign
       - :name: unassign
+  :firmwares:
+    :description: Firmwares
+    :options:
+    - :collection
+    - :subcollection
```

Comment:
This should be the PhysicalServer collection.  The PhysicalServer collection is defined in PR https://github.com/ManageIQ/manageiq/pull/14028.